src/test/MvcTestApp/Startup.cs

```diff
@@ -41,13 +70,21 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 app.UseExceptionHandler("/Home/Error");
             }
-            
+
             app.UseStaticFiles();
 
             app.UseLoupeCookies();
 
             app.UseRouting();
 
+            /*
+             * If you have added CORS support in ConfigureServices above
+             * you need to use that policy here
+             
+            app.UseCors("AllowOrigin");
```

Comment:
In .NET Core 3.1 and later, you can also configure CORS on the endpoints, which would look something like this:

```csharp
endpoints.MapLoupeClientLogger().RequireCors("AllowOrigin");
```

Which is useful if you want CORS support on the logging but not on controllers etc.

Note that this method does not yet support automatic preflight OPTIONS requests: https://docs.microsoft.com/en-us/aspnet/core/security/cors?view=aspnetcore-3.1#enable-cors-with-endpoint-routing

Reply:
I did wonder about using DefaultPolicy. What's your view on what we should put here? Happy to change it if you feels that's more appropriate. My aim is to really just get them going with the sample, and we can always add a link in to the docs for CORS configuration.